railties/lib/rails/commands/server/server_command.rb

```diff
@@ -112,12 +112,18 @@ class ServerCommand < Base # :nodoc:
         desc: "Specifies the PID file."
       class_option "dev-caching", aliases: "-C", type: :boolean, default: nil,
         desc: "Specifies whether to perform caching in development."
+      class_option "log-stdout", aliases: "-l", type: :boolean, default: nil,
+        desc: "Specifies whether to output Rails' log to std out (true = always, false = never, default = in development). Running as a Daemon always disables output."
```

Comment:
There's something about having a boolean mean 3 things that seems off to me. I'd think we'd only want `--log-to-stdout` (e.g. no `--no-log-to-stdout`) which just overrides whatever logging setup is there. Perhaps we should just honor it despite `:daemon` as well? Or throw an exception if both are enabled?

Reply:
Changed it so that it raises in case of being daemonized and logging enabled.

It is still possible to do `--log-to-stdout false` and "--no-log-to-stoud" afaik though, which is OK imo.